### PR TITLE
Add centralised logging and metrics

### PR DESCRIPTION
**Pull request by samleeflang**
Some minor changes to the Kafka processors

### CODE REVIEW

File: installation.md [hunk 1]
```diff
@@ -2,12 +2,12 @@
 
 ## Make sure infrastructure is deployed
 First deploy all infrastructure necessary for the applications using TerraFrom,see the [infrastructure repo](https://github.com/DiSSCo/dissco-core-infrastructure). 
-If this was de deployed succesfully you should now have a document-db, rds postgres database and a kubernetes cluster. 
+If this was de deployed successfully you should now have a document-db, rds postgres database and a kubernetes cluster.
```
Comment (TomDijkema) [inline installation.md:5]: Is the 'de' before 'deployed' intentional?

File: test/digital-media-object/digital-media-object-processor-kafka.yaml [hunk 2]
```diff
@@ -82,10 +80,10 @@ spec:
               value: https://login-demo.dissco.eu/auth/realms/dissco/protocol/openid-connect/token
           resources:
             requests:
-              memory: "500Mi"
-              cpu: "200m"
+              memory: "1Gi"
```
Comment (TomDijkema) [inline test/digital-media-object/digital-media-object-processor-kafka.yaml:83]: Doubling it up I see, was the digital media processor not able to handle it at half the capacity?

File: test/digital-media-object/digital-media-object-processor-kafka.yaml [hunk 3]
```diff
@@ -98,7 +96,7 @@ spec:
   scaleTargetRef:
     name: dissco-digital-media-object-processor-kafka
   minReplicaCount: 0
-  maxReplicaCount:  5
+  maxReplicaCount:  1
   triggers:
```
Comment (southeo) [inline test/digital-media-object/digital-media-object-processor-kafka.yaml:100]: why reduce the maxReplicaCount?

**Review by southeo — APPROVED** [review]
:deciduous_tree: :evergreen_tree: 
... get it? Cuz logging?

File: test/digital-media-object/digital-media-object-processor-kafka.yaml (hunk 3 above)
Comment (samleeflang) [reply test/digital-media-object/digital-media-object-processor-kafka.yaml:100]: It is a bit enthusiastic :smile:  it pulled down the cluster and it overloaded the handle-api. For the time being one will be enough. We need auto-scaling on the cluster and on the handle-api (and prob database) if we want to kind of horizontal scalability.

**Pull request merged**